mindsdb/api/http/namespaces/config.py

```diff
@@ -135,14 +136,20 @@ def put(self, name):
                 handler_type=handler_type,
                 connection_data=params
             )
-            handler = info['handler']
 
             status = handler.check_connection()
             if temp_dir is not None:
                 shutil.rmtree(temp_dir)
 
             resp = status.to_json()
-            resp['integration_id'] = info['args']['integration_id']
+            if status.success and hasattr(handler, 'handler_storage'):
+                # attach storage if exists
+                export = handler.handler_storage.export_files()
```

Comment:
do we really need this for any handler with files?

Reply:
if handler hasn't files export will be null
and on frontend output export is used only after auth window. in other cases frontend doesn't pass export to backend

Reply:
- not only gmail handler has files. For example mysql also may have it (ssl certs).
 - `last_modified.txt` file will be in each handler dir